schemas/ui/v1.0/ui.schema

```diff
@@ -50,6 +50,15 @@
                         "type": "string"
                     }
                 },
+                "intellisenseScopes": {
+                    "title": "Intellisense scopes",
+                    "description": "An array of valid scopes to help Intellisense return the correct type of results.",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "examples": ["expressions","user-variables"]
```

Comment:
```suggestion
                    "examples": [["expressions","user-variables"]]
```

This needs to be an array of arrays instead.

Reply:
good catch!

Reply:
I'm confused.  That is not what you have modeled in the schema.  Why an array of arrays?  And if so the schema and description should reflect that.  The items should be of type array with items of string.

---
In reply to: [492409522](https://github.com/microsoft/botframework-sdk/pull/6047#discussion_r492409522) [](ancestors = 492409522)